docs/test_types/TestStructures/BlockchainTests/block.rst

```diff
@@ -40,19 +40,58 @@ Valid Block Section
             }
 
-Contains the block information that is going to be imported into test chain. The key field here is **rlp**. All testers take `rlp` field data and import it on top of genesis that was described in test by **genesisRLP**
+Contains the block information that is going to be imported into test chain. 
+The key field here is **rlp**. All testers take `rlp` field data and import it on 
+top of genesis that was described in test by **genesisRLP**
 
-Fields "blockHeader`, `transactions`, `uncleHeaders` describe the content of `rlp` for humans to read. it's content **MUST** be equal to `rlp` field binary. All testers use **rlp** field data to import it into the test chain. 
+Fields `blockHeader`, `transactions`, `uncleHeaders` describe the content of 
+`rlp` for humans to read. Its content **MUST** be equal to `rlp` field binary. 
+All testers use **rlp** field data to import it into the test chain. 
 
 Fields `chainname`, `blocknumber` are irrelevant for the test client.
 
+
 * all fields are 0x prefixed HEX
 * 0 values are 0x00 rounded
 * empty extraData 0x
 * transaction creation `to` is ""
+* the `transaction` 
+
+
+
+
+Valid Blocks with Invalid Transactions
+--------------------------------------
+
+::
+
+                "transactionSequence" : [
+                    {
+                        "valid" : "false",
+                        "rawBytes" : "0x02f87001018502540be4008502540be40086246139ca800094cc...
+                        "exception" : "TR_NoFunds"
+                    },
+                    {
+                        "valid" : "true",
+                        "rawBytes" : "0x02f87001018502540be4008502540be40086246139ca800094cc...
+                    }
+                ]
+
+Blocks that include invalid transactions have an additional field, 
```

Comment:
Put a comment here.
That this is done for those to test transaction rejection. What we do here is

Lets pretend that the block have a given block header 
And then transactions from transaction sequence come in a given order. Those that are invalid must be rejected. 

Before this was being done only when filling the test. Leaving the block empty. I put it for custom test runners in this format  so they can test transaction rejection cases.